core/src/main/java/bio/terra/pearl/core/service/export/formatters/module/EnrolleeFormatter.java

```diff
@@ -11,11 +10,16 @@
 
 public class EnrolleeFormatter extends BeanModuleFormatter<Enrollee> {
     public static final String ENROLLEE_MODULE_NAME = "enrollee";
-    public static final List<String> INCLUDED_PROPERTIES = List.of("shortcode", "consented", "createdAt");
+    public static final List<String> INCLUDED_PROPERTIES = List.of(
+            "shortcode", "consented", "createdAt"
+    );
 
     public EnrolleeFormatter(ExportOptions exportOptions) {
         itemFormatters = INCLUDED_PROPERTIES.stream().map(propName -> new PropertyItemFormatter<Enrollee>(propName, Enrollee.class))
                 .collect(Collectors.toList());
+        if (exportOptions.includeProxiesAsRows()) {
```

Comment:
Let's always include this column.  Even though most of the time it will always just be "true" it's a useful reminder